apps/web/components/v2/eventtype/EventAppsTab.tsx

```diff
@@ -70,8 +71,20 @@ export const EventAppsTab = ({
 
   const { t } = useLocale();
 
+  const installedApps = [hasPaymentIntegration, hasGiphyIntegration].filter(Boolean).length;
+
+  if (installedApps === 0) {
+    return (
+      <EmptyScreen
+        Icon={Icon.FiGrid}
+        headline={t("empty_installed_apps_headline")}
+        description={t("empty_installed_apps_description")}
+        buttonRaw={<Button href="/apps">{t("empty_installed_apps_button")} </Button>}
```

Comment:
it would be nice if we save event type settings here before routing

Reply:
Will pick this up in a separate PR cause I agree :) - modal popop warning the user too